https://ant.apache.org/manual/Tasks/resourcecount.html

Display or set a property containing the size of a nested Resource Collection.
Can also be used as a condition. Since Apache Ant 1.7

--- apache-ant-1.7.1/build.xml~	2008-06-27 08:05:23.000000000 +0300
+++ apache-ant-1.7.1/build.xml	2012-05-08 08:09:17.536508489 +0300
@@ -501,12 +501,14 @@
 
     <condition property="tests.are.on.system.classpath">
       <or>
+		  <!--
         <resourcecount count="1">
           <intersect>
             <path path="${java.class.path}"/>
             <file file="${build.tests}"/>
           </intersect>
         </resourcecount>
+		-->
         <istrue value="${junit.fork}"/>
       </or>
     </condition>
